vite.config.ts

```diff
@@ -74,7 +74,7 @@ const jsToBottomNoModule = () => {
 export default defineConfig(({ mode }) => {
     process.env = { ...process.env, ...loadEnv(mode, process.cwd(), '') }
     const baseConfig = {
-        base: '/dashboard/',
+        base: '/dashboard',
```

Comment:
Do we require this change?

Reply:
yes. it is causing // after dashboard on local. May be we should also change this in .env file